modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -510,8 +531,13 @@ void ParallelCoordinates::drawHandles(size2_t size) {
             sliderWidgetRenderer_.setUIColor(handleColor_);
         }
 
-        axis.sliderWidget->render(ivec2{ap.first} - ivec2{handleSize_.get(), handleWidth} / 2,
-                                  size);
+        if (axis.pcp->invertRange) {
+            axis.sliderWidget->render(ivec2{ap.first} - ivec2{handleSize_.get(), handleWidth} / 2,
```

Comment:
Identical?

Reply:
Yes, that was a left-over. If statement was removed.